### PR TITLE
change the padding to take into account the border

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
This PR changes the padding for the `input` this takes into account the border for the inputs. The size of the `textinput` without being wrapped in a `formfield` is `38px` with wrapping it in a `formfield` then it uses the `textinput` plain so without the border and is then the correct size.
#### What testing has been done on this PR?
in the design system site
#### Any background context you want to provide?
Below is a picture of the sizes for `button` the padding is off by one to account for the border. I believe we need to do the same for the `inputs` as well. Since `Select` and  `Menu` are made from `button` their sizes are correct. 
Screen Shot 2021-09-08 at 3 59 25 PM

With this change to the padding you can see that they both will match up correctly. 
Screen Shot 2021-09-08 at 4 02 00 PM

#### What are the relevant issues?
Closes https://github.com/grommet/hpe-design-system/issues/1778
#### Is this change backward compatible or could it be a breaking change for the official HPE theme?
backward compatible

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -164,8 +164,8 @@ export const hpe = deepFreeze({
         weight: 500,
       },
       padding: {
-        horizontal: 'small',
-        vertical: 'xsmall',
+        horizontal: '11px',
```
Comment (taysea) [inline src/js/index.js:167]:
```suggestion
        horizontal: '11px', // equivalent to 'small' when combined with 1px border
```

File: src/js/index.js [hunk 2]
```diff
@@ -164,8 +164,8 @@ export const hpe = deepFreeze({
...
+        vertical: '5px',
```
Comment (taysea) [inline src/js/index.js:168]:
```suggestion
        vertical: '5px', // equivalent to 'small' when combined with 1px border
```

File: src/js/index.js [hunk 3]
```diff
@@ -164,8 +164,8 @@ export const hpe = deepFreeze({
         weight: 500,
       },
       padding: {
-        horizontal: 'small',
-        vertical: 'xsmall',
+        horizontal: '11px',
+        vertical: '5px', // equivalent to 'small' when combined with 1px border
```
Comment (taysea) [inline src/js/index.js:168]:
```suggestion
        vertical: '5px', // equivalent to 'xsmall' when combined with 1px border
```

**Review by taysea — APPROVED** [review]

**Review by taysea — COMMENTED** [review]
Thinking we should put a PR up on design-system-site just to check if any snapshot changes exist that we might not have caught in manual testing, but otherwise looks good.

**Comment by britt6612** [comment]
> Thinking we should put a PR up on design-system-site just to check if any snapshot changes exist that we might not have caught in manual testing, but otherwise looks good.

oh yes good thought Ill do that!

**Comment by taysea** [comment]
Theme changes on local deployment of site look good. Good to go here!

**Review by ericsoderberghp — APPROVED** [review]

**Pull request merged**